pytheus/metrics.py

```diff
@@ -48,15 +53,21 @@ def _validate_labels(self, labels: Sequence[str]):
 
 # class or function based creation ?
 class Metric:
-
     def __init__(
         self,
         collector: MetricCollector,
         labels: Labels | None = None,
+        backend_class: Optional[Backend] = None,
```

Comment:
to be moved like described above

Reply:
89873829500450ebc594a994fc28443e16ed6a4b